src/hooks/index.ts

```diff
@@ -870,9 +906,19 @@ export const useResync = (wallet: YoroiWallet, options?: UseMutationOptions<void
   }
 }
 
-export const useNfts = (wallet: YoroiWallet) => {
+export const useNfts = (wallet: YoroiWallet, {search}: {search?: string} = {}) => {
```

Comment:
can we store nfts in the wallet instead of reacy-query?
then in the component use something like
```
useNfts({ wallet }) // subscribes to nfts/utxos
const nfts = wallet.balances.nfts
```

Reply:
What are the benefits here and is this something we want to change in this PR?